assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/steps/PlaceTagsStep.js

```diff
@@ -37,6 +41,8 @@ import {
 	ENUM_AD_BLOCKING_RECOVERY_SETUP_STEP,
 	MODULES_ADSENSE,
 } from '../../../../datastore/constants';
+import Link from '../../../../../../components/Link';
+import { CORE_SITE } from '../../../../../../googlesitekit/datastore/site/constants';
```

Comment:
The same request applies here, could you move this below the other imported constants?

Reply:
Updated ✅